packages/plugin-doc-coverage/src/lib/runner/__snapshots__/doc-processer.integration.test.ts.snap

```diff
@@ -0,0 +1,153 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`processDocCoverage > should succesfully get the right number of ts files 1`] = `
+{
+  "classes": {
+    "coverage": 100,
+    "issues": [],
+    "nodesCount": 1,
+  },
+  "enums": {
+    "coverage": 100,
+    "issues": [],
+    "nodesCount": 0,
+  },
+  "functions": {
+    "coverage": 33.33,
+    "issues": [
+      {
+        "file": "/home/alejandro/dev/code-pushup-cli/packages/plugin-doc-coverage/mocks/fixtures/angular/app.component.spec.ts",
+        "line": 1,
+        "name": "notRealisticFunction",
+        "type": "functions",
+      },
+      {
+        "file": "/home/alejandro/dev/code-pushup-cli/packages/plugin-doc-coverage/mocks/fixtures/angular/map-event.function.ts",
```

Comment:
This code includes your private path. I wonder how this can pass the test in CI. 

You can use our helper to strip path data from the snapshot. 

`import {  omitVariableReportData} from '@code-pushup/test-utils';`

Reply:
oh shiet

Reply:
It kinda does not let me use it unless I pass an any... The parameters are vastly different and when I have seen use cases of that method, it actually means to use `Reports`. Anyway this integration snapshot does not exists anymore.